### PR TITLE
Feat/css colors helpers

### PR DESCRIPTION
**Pull request by rustem-nasyrov**
Related issue: #1222 
closes https://github.com/epicmaxco/vuestic-ui/issues/764 #2169

image

## Description
Added customized classes for the given colors configuration.
* Created `colorsCustomClasses` plugin for the Vuestic-UI.
* Added description in the docs, for the both translations.
* Added examples of usage.

## Types of changes
- [x] New feature (non-breaking change which adds functionality)
- [x] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

### CODE REVIEW

File: packages/docs/src/locales/ru/ru.json [hunk 1]
```diff
@@ -3346,7 +3353,8 @@
       "readMore": "Подробнее",
       "icons": "Конфигурация иконок",
       "components": "Конфигурация компонентов",
-      "colors": "Конфигурация цветов"
+      "colors": "Конфигурация цветов",
+      "colorsCustomClasses": "Custom classes for the colors"
```
Comment (aluarius) [inline packages/docs/src/locales/ru/ru.json:3357]: Missed translation.

**Comment by asvae** [comment]
Thanks!

Implementation wise looks good, but I hope to discuss what place would this solution take in context of vuestic-ui.

**Review by aluarius — APPROVED** [review]

**Comment by m0ksem** [comment]
Let's discuss prerender and some caching here... It looks good, but something is missing here I believe...

**Comment by rustem-nasyrov** [comment]
> Let's discuss prerender and some caching here... It looks good, but something is missing here I believe...

Yeah, sure.

**Pull request closed without merging**